eth/v1alpha1/validator.proto

```diff
@@ -292,6 +295,26 @@ enum ValidatorStatus {
   EXITED = 6;
 }
 
+message ValidatorStatusMetadata {
+    // The corresponding validator status.
+    ValidatorStatus status = 1;
+
+    // The block number of the Ethereum proof-of-work chain
+    // where the deposit for the validator was included.
+   uint64 eth1_deposit_block_numbers = 2;
+
+    // The slot in the beacon chain in which the validator's
+    // deposit was included in a block.
+   uint64 deposit_inclusion_slots = 3;
+
+    // The epoch in the beacon chain in which the validator
+    // is determined as active.
+   uint64 activation_epochs = 4;
+
+    // The position in the activation queue of pending validators.
+   uint64 position_in_activation_queues = 5;
+}
+
 message ValidatorStatusResponse {
     // The corresponding validator status.
     ValidatorStatus status = 1;
```

Comment:
Mark these as deprecated

Reply:
As discussed on [Discord](https://discordapp.com/channels/476244492043812875/667763844365680650/707725202188009484). These fields are used here: https://github.com/prysmaticlabs/ethereumapis/blob/df460bd3d84be4ff3df0658395c7dc9d2a7e7b3d/eth/v1alpha1/validator.proto#L239

So we won't deprecate them right now.